.github/workflows/npm-cd.yml

```diff
@@ -21,7 +21,7 @@ jobs:
                           ARCH: x64,
                           TARGET: x86_64-unknown-linux-gnu,
                           NPM_PUBLISH: true,
-                          PYPI_PUBLISH: true,
+                          PYPI_PUBLISH: false,
```

Comment:
Since you split the CD to separate workflows for npm and pypi, we can completely remove the NPM_PUBLISH and PYPI_PUBLISH flags

Reply:
If we want to save the NPM_PUBLISH flag so we can easily test publishing only to specific platforms, we can remove only the PYPY_PUBLISH flag from npm-cd and the NPM_PUBLISH from pypi-cd